spec/unit/indirector/rest_spec.rb

```diff
@@ -353,7 +353,7 @@ def save_request(key, instance, options={})
       it "calls get on the connection" do
         request = find_request('foo bar')
 
-        connection.expects(:get).with("#{url_prefix}/test_model/foo%20bar?environment=production&", anything).returns(mock_response('200', 'response body'))
+        connection.expects(:get).with("#{url_prefix}/test_model/foo+bar?environment=production&", anything).returns(mock_response('200', 'response body'))
```

Comment:
Upon further discussion, there *should* be a distinction for path vs query - thanks for pointing that out @rlinehan - some discussion at http://stackoverflow.com/a/29948396 and https://stackoverflow.com/questions/2824126/whats-the-difference-between-uri-escape-and-cgi-escape

Looks like I need to take a closer look at this PR to make sure I'm doing the best we can do.